tests/bluetooth/tester/src/btp_aics.c

```diff
@@ -539,17 +566,84 @@ static void aics_description_cb(struct bt_aics *inst, int err, char *description
 	uint8_t data_len = strlen(description);
 
 	bt_aics_client_conn_get(inst, &conn);
-	btp_send_aics_description_ev(conn, data_len, description);
+	btp_send_aics_description_ev(conn, err, data_len, description);
 
 	LOG_DBG("AICS description callback (%d)", err);
 }
 
+static void aics_set_gain_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_SET_GAIN;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
+
+	LOG_DBG("AICS set gain cb (%d)", err);
+}
+
+static void aics_mute_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_MUTE;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
+
+	LOG_DBG("AICS mute cb (%d)", err);
+}
+
+static void aics_unmute_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_UNMUTE;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
+
+	LOG_DBG("AICS unmute cb (%d)", err);
+}
+
+static void aics_set_man_gain_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_MAN_GAIN_SET;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
+
+	LOG_DBG("AICS set manual gain cb (%d)", err);
+}
+
+static void aics_set_auto_gain_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_AUTO_GAIN_SET;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
```

Comment:
```suggestion
	bt_aics_client_conn_get(inst, &conn);

	btp_send_aics_procedure_ev(conn, err, BTP_AICS_AUTO_GAIN_SET);
```

These could be a bit more simple, but they are also OK as is

Reply:
Fixed